scheduler/src/cook/mesos/api.clj

```diff
@@ -2041,7 +2040,8 @@
                                 ::start-ms (parse-long-default start-ms nil)
                                 ::limit (parse-int-default limit Integer/MAX_VALUE)
                                 ::end-ms (parse-long-default end-ms (System/currentTimeMillis))
-                                ::name-filter-fn (name-filter-str->name-filter-fn name)}]
+                                ::name-filter-fn (name-filter-str->name-filter-fn name)
+                                ::include-custom-executor? (Boolean/parseBoolean include-custom-executor)}]
```

Comment:
Nitpick: It's probably better to use `Boolean/valueOf` here so as to avoid the implicit auto-boxing. (`valueOf` returns a `Boolean` object, whereas `parseBoolean` returns a `boolean` primitive).

Reply:
Done